### PR TITLE
update angular 2 example

### PR DESCRIPTION
**Pull request by kasperpeulen**
#### update angular 2 example
* update to latest beta
* removed library tags
* added environment constraint
* added platform directives and pipes

### CODE REVIEW

**Comment by devoncarew** [comment]
Thanks! lgtm for me; @kevmoo is familiar with the angular sample and should have a chance to look that over -

File: templates/web-angular/web/main.dart [hunk 1]
```diff
@@ -1,8 +1,6 @@
 // Copyright (c) __year__, __author__. All rights reserved. Use of this source code
 // is governed by a BSD-style license that can be found in the LICENSE file.
 
-library __projectName__.web;
-
 import 'package:angular2/bootstrap.dart';
 
 import 'package:__projectName__/app_component.dart';
```
Comment (kwalrath) [inline templates/web-angular/web/main.dart]: Should we just use `import 'app_component.dart'` here? (Not a big deal, either way, to me.)

**Comment by kwalrath** [comment]
LGTM

**Comment by kwalrath** [comment]
We don't need to wait for @kevmoo to review, IMO.

**Comment by devoncarew** [comment]
Cool, merging -

**Comment by kasperpeulen** [comment]
thanks !

On Sun, Jan 31, 2016 at 9:57 PM, Devon Carew <notifications@github.com>
wrote:

> Merged #298 <https://github.com/google/stagehand/pull/298>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/google/stagehand/pull/298#event-533318951>.
>



-- 
Kasper

**Pull request merged**